### PR TITLE
Generate standalone plugin with init:plugin command - Closes #6089 

### LINKED ISSUE #6089: Create standalone plugin bootstrapping generator
### Description

This generator will bootstrap a standalone plugin with all its dependencies. The interface of the generator should be:

```
lisk init:plugin <alias> <path>
```

### Motivation

User should be able to generate a skeleton of the plugin quickly. It would be basically an npm package with plugin class exported from the package. 

### Acceptance Criteria
- Current directory will be used as path if not provided 
- The plugin class should be written under `src` directory. 
- The plugin should be exported from the main script file 
- If there is no npm package in the path then it should be created 
- It should also generate the relevant test skeleton. 

### Additional Information
- The plugin source code should include load and unload methods
- The rest of plugin require data should be populated from `package.json` file. 
- It should have a sample array of events but commented 
- It should have a sample actions object but commented 
- For reference we can use the skeleton of framework plugins

### PR DESCRIPTION
**Pull request by mitsuaki-u**
### What was the problem?

This PR resolves #6089 

### How was it solved?

- Added init:plugin command
- Added init:plugin base generator
- Added init:plugin generator
- Updated types in all files for consistency
- Added templates for standalone plugin
- Added package.json file
- Added all configuration files
- Added scripts

### How was it tested?

1. Run `yarn link` in `lisk-sdk/sdk` folder

2. Run generate standalone plugin command from commander: `./bin run init:plugin pluginName ~/path/to/plugin`

    **Note: If you run this command without the path argument e.g. ~/path/to/plugin", it will create git files in the current working commander directory and ask for overwrite. In production the user will not be running this command from the commander directory.**

3. Run `yarn link lisk-sdk` in generated plugin folder

4. Check output

### CODE REVIEW

File: commander/src/commands/init/plugin.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+import BaseBootstrapCommand from '../../base_bootstrap_command';
+
+export default class PluginCommand extends BaseBootstrapCommand {
+	static description = 'Creates custom plugin.';
```
Comment (Incede) [inline commander/src/commands/init/plugin.ts:20]:
```suggestion
	static description = 'Creates custom plugin as a standalone package containing all of it's dependencies.';
```

File: commander/src/commands/init/plugin.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+	static examples = ['generate:plugin my-plugin', 'generate:plugin my-plugin ./path'];
```
Comment (Incede) [inline commander/src/commands/init/plugin.ts:21]:
```suggestion
	static examples = ['init:plugin my-plugin', 'init:plugin my-plugin ./path'];
```

File: commander/src/bootstrapping/templates/lisk-template-ts/generators/init_plugin_generator.ts [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+				type: 'input',
+				name: 'name',
+				message: 'Name of plugin',
+				default: this._alias,
+			},
+			{
+				type: 'input',
+				name: 'description',
+				message: 'Description of plugin',
+				default: 'Lisk-SDK Application',
```
Comment (ishantiw) [inline commander/src/bootstrapping/templates/lisk-template-ts/generators/init_plugin_generator.ts:69]: is this going to be different? like a standalone plugin app?

File: commander/src/bootstrapping/templates/lisk-template-ts/templates/init_plugin/package.json [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+	"author": "<%= author %>",
+	"license": "<%= license %>",
+	"keywords": [],
+	"homepage": "",
+	"repository": {
+		"type": "git",
+		"url": ""
+	},
+	"bugs": {
+		"url": ""
```
Comment (ishantiw) [inline commander/src/bootstrapping/templates/lisk-template-ts/templates/init_plugin/package.json:14]: it shows warning `String is not a URI: URI expected.` should we remove it?

File: commander/src/bootstrapping/templates/lisk-template-ts/templates/init_plugin/test/unit/plugin.spec.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
+// import * as plugins from '../../src/plugin';
+
+describe('<%= className %>Plugin', () => {
```
Comment (ishantiw) [inline commander/src/bootstrapping/templates/lisk-template-ts/templates/init_plugin/test/unit/plugin.spec.ts:3]:
```suggestion
describe('<%= className %>', () => {
```
class name already contains Plugin suffix

File: commander/src/bootstrapping/templates/lisk-template-ts/templates/plugin/test/unit/plugins/plugin.spec.ts [hunk 6]
```diff
@@ -1,24 +1,24 @@
 // import * as plugins from '../../../src/app/plugins/<%= alias %>';
 
 describe('<%= className %>Plugin', () => {
```
Comment (ishantiw) [inline commander/src/bootstrapping/templates/lisk-template-ts/templates/plugin/test/unit/plugins/plugin.spec.ts:3]:
```suggestion
describe('<%= className %>', () => {
```

**Review by ishantiw — COMMENTED** [review]
Overall LGTM, just a few comments

File: commander/src/bootstrapping/templates/lisk-template-ts/generators/init_plugin_generator.ts (hunk 3 above)
Comment (mitsuaki-u) [reply commander/src/bootstrapping/templates/lisk-template-ts/generators/init_plugin_generator.ts:69]: It is the same default response as `init` command

**Review by Incede — APPROVED** [review]

**Review by ishantiw — APPROVED** [review]

**Pull request merged**